src/flows/prospecting/prospecting_orchestration_flow.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+import boto3
+import boto3.session
+from pendulum import now as pd_now
+from prefect import Flow, task, unmapped
+from prefect.executors import DaskExecutor
+from prefect.tasks.aws import StepActivate
+
+from utils.config import (
+    ENVIRONMENT,
+    PROSPECTING_PROSPECTS_FLOWS,
+    PROSPECTING_TRAINING_FLOWS,
+)
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+# List of step function workflows to execute.
+STEP_FUNCTION_TRAINING_FLOWS = PROSPECTING_TRAINING_FLOWS[ENVIRONMENT]
+STEP_FUNCTION_PROSPECTS_FLOWS = PROSPECTING_PROSPECTS_FLOWS[ENVIRONMENT]
+
+# Base input to pass to the step function execution
+BASE_EXECUTION_INPUT = json.dumps(
+    {
+        "Parameters": "{}",
+    }
+)
+
+
+@task()
+def create_sfn_arns(sfn_flow_names: list) -> list:
+    # Create your session as suggested for threading
+    init_session = boto3.session.Session()
+    # get region name and account id for return list
+    region_name = init_session.region_name
+    sts = init_session.client("sts")
+    account_id = sts.get_caller_identity()["Account"]
+    return [
+        f"arn:aws:states:{region_name}:{account_id}:stateMachine:{x}"
+        for x in sfn_flow_names
+    ]
+
+
+@task()
+def create_sfn_execution_names(sfn_flow_names: list) -> list:
+    return [f"{x}-{pd_now('UTC').int_timestamp}" for x in sfn_flow_names]
+
+
+with Flow(
+    FLOW_NAME, schedule=get_interval_schedule(minutes=120), executor=DaskExecutor()
```

Comment:
```suggestion
    FLOW_NAME, schedule=get_interval_schedule(minutes=120), executor=LocalDaskExecutor()
```

Reply:
Optional optimization: Use a cron schedule to try to (somewhat) align the execution time of this flow with the completion time of the Dbt job.

Reply:
ah yes.....I will do that

Reply:
change made